docs/Examples/Planets/Get-Planet.Tests.ps1

```diff
@@ -101,11 +106,7 @@ Describe 'Get-Planet' {
             $planets = Get-Planet -Name $Filter
             # We validate that the returned name is equal to $Expected.
             # That is Neptune, in our second test.
-            $planets | Select -ExpandProperty Name | Should -Be $Expected
```

Comment:
Run lint scan, Select should be Select-Object

Reply:
Is this comment still relevant though, given that we are just performing a "Select speciific property" on the `$planets` object. (Which is valid syntax in PS3+ (such as $planets.Name). See also the comment in original code on line 108.

Reply:
Aliases are valid syntax, and Select is a valid alias identified directly in documentation (but not Get-Alias for some reason?).  It is picked up as a PSAvoidUsingCmdletAliases warning with PSScriptAnalyzer.  

IMO scripts shouldn't have any warnings as part of lint checks / scans, or if they do there should be some exclusions file in the project with details why the issue is being excluded.  There's 686 warnings in the Pester repo currently.  Some warnings might be invalid due to ways that Pester works, others could be fixed to improve code quality and readability.

```powershell
Invoke-ScriptAnalyzer . -Recurse | Measure-Object -Line
```

I saw this rule picked up in a few scripts I wrote when using VSCode's PSScriptAnalyzer plugin.  As part of build pipeline for Pester, PSScriptAnalyzer should pick up these warnings.

![image](https://user-images.githubusercontent.com/7253192/88271749-9fbf3580-cca5-11ea-8ed4-8a40c5a535e0.png)

Some details on alias issues.
https://martin77s.wordpress.com/2015/02/04/dont-use-aliases-in-scripts/
https://tommymaynard.com/cmdlet-and-function-alias-best-practice-2015/

I actually disagree with some of this, less is more sometimes and Select = Select-Object in my mind too.  

Since the example is being updated, why not fix the 3 linting issues?

Reply:
I do agree that the aliases make thing more hard to read - and not only that but also less performant. 

Moved to using the non-alias variants, select was already removed, 
but also adjusted the `foreach` -> `ForEach-Object` and `where` -> `Where-Object`.